eng/common/pipelines/templates/steps/verify-changelog.yml

```diff
@@ -2,7 +2,7 @@ parameters:
 - name: PackageName
   type: string
   default: 'not-specified'
-- name: ServiceName
+- name: ServiceDirectory
```

Comment:
This is technically a breaking change and will cause a break for any consumers in this repo, in particular with the release. Please verify with a template release. 

This change needs to go along with https://github.com/Azure/azure-sdk-for-net/pull/13783/files, somehow we need to coordinate these 2 changes to test and ensure we don't break anything.